src/gui/widgets/EffectView.cpp

```diff
@@ -111,7 +112,7 @@ EffectView::EffectView( Effect * _model, QWidget * _parent ) :
 		m_controlView = effect()->controls()->createView();
 		if( m_controlView )
 		{
-			if( !(m_controlView->windowTitle() == "Peak Controller") )
+			if( dynamic_cast<PeakControllerEffectControlDialog*>( m_controlView ) == NULL )
```

Comment:
I had to remove the Q_OBJECT macro from PeakControllerEffectDialog otherwise I get an linker error. Actually the Q_OBJECT macro isn't needed because there is no Signal/Slot stuff in this class. and everything works fine. But eventually someone is looking at this.